### ISSUE TITLE
recommending using a table for a 2 columns presentation is bad

### ISSUE THREAD

**Issue by Erwan-le-Gall** [suitenumerique/docs#1862]
## Bug Report

**Problematic behavior**
In this documentation page https://docs.numerique.gouv.fr/docs/e314316e-0ec7-49bd-bc2c-1d60cf34a119/ it is recommended to use a table to put 2 contents side by side.

It is generally a bad recommendation from the accessibility point of view: this invalidate an accessibility criteria which specify that no fonctionality should be diverted to obtain a presentation effect. Code should be semantic. Tables are data tables only and should not be used for anything else.

NB Microsoft Word allow to create a presentation table by removing its style.

By the way **« en-tête » is a masculine word in French**: « un en-tête ».

**Expected behavior/code**
No help should induce users to create inaccessible documents.

**Steps to Reproduce**
1. Play the tutorial video.

**Environment**
- Docs version: unknown. How can I find this as a user?
- Instance url: https://docs.numerique.gouv.fr/docs/

**Possible Solution**
Remove the video (which is inaccessible to blind users, by the way).

**Additional context/Screenshots**
Accessibility requirement.

**Comment by virgile-dev** [comment]
Thanks I corrected the gender of en-tête.
Regarding the rest, columns is a feature that we enabled for users.
@cyberbaloo if needed please open accessibility tickets regarding this but we won't remove it.
The video is there to provide a tip for users.
